sql/abilities.sql

```diff
@@ -217,7 +217,7 @@ INSERT INTO `abilities` VALUES (177,'curing_waltz_iv',19,70,27,12,217,306,0,9,20
 INSERT INTO `abilities` VALUES (178,'healing_waltz',19,35,27,8,215,0,0,10,2000,0,14,20.0,0,1,300,0,0,'WOTG');
 INSERT INTO `abilities` VALUES (179,'divine_waltz',19,25,27,13,225,306,0,11,2000,0,14,20.0,1,0,0,0,0,'WOTG');
 INSERT INTO `abilities` VALUES (180,'spectral_jig',19,25,1,30,218,0,0,12,2000,0,14,20.0,0,1,300,0,0,'WOTG');
-INSERT INTO `abilities` VALUES (181,'chocobo_jig',19,55,1,60,218,126,0,13,2000,0,14,20.0,0,1,300,0,0,'WOTG');
+INSERT INTO `abilities` VALUES (181,'chocobo_jig',19,55,1,60,218,126,0,13,2000,0,14,20.0,0,1,300,0,0,'SOA');
```

Comment:
I am pretty sure that Chocobo Jig was WOTG, because I remember having it, and I never played SOA...
https://www.bg-wiki.com/index.php?title=Chocobo_Jig&oldid=47565

Reply:
Fair point, reverted 👍